packages/nimble-blazor/NimbleBlazor/wwwroot/NimbleBlazor.lib.module.js

```diff
@@ -118,6 +118,17 @@ export function afterStarted(Blazor) {
             };
         }
     });
+    // Used by NimbleTable.razor
+    Blazor.registerCustomEventType('nimbletablerowexpandtoggle', {
+        browserEventName: 'row-expand-toggle',
```

Comment:
I certainly may have missed it but is there an acceptance / playwright test for this event?

Reply:
There wasn't one, but now I've added it.

Reply:
love it 👍